website/src/utils/templateProcessor.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Process a URL template by replacing placeholders with values
+ */
+export function processTemplate(template: string, params: Record<string, string>) {
```

Comment:
```suggestion
 * Process a URL template by replacing placeholders with values
 */
export function processTemplate(template: string, placeholders: Record<string, string>) {
```

Or `placeholdersMap`? Or `placeholdersAndValues`? I think that referring to "placeholders" is clearer, especially because the function documentation refers to "placeholders"